src/nsmgr/internal/pkg/values/grpc_url_value.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2020 Cisco and/or its affiliates.
```

Comment:
Please deprecate in favor of: https://github.com/networkservicemesh/sdk/blob/master/pkg/tools/flags/url_value.go